# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1360475931 18000
#      Sun Feb 10 00:58:51 2013 -0500
# Node ID f124f66af2739221d88281868f4a8c95a0125039
# Parent  6076458ddf988a2d12656fd0d1e03c9de15c0f0f
update to new version of gnulib

diff --git a/.hgsubstate b/.hgsubstate
--- a/.hgsubstate
+++ b/.hgsubstate
@@ -1,1 +1,1 @@
-e2f98cc246bdb12f2d35366042d4aab61c3d54fe gnulib-hg
+fea72d94801364b1bde7cdf83237a49fe4db508a gnulib-hg
